bftengine/include/bftengine/ReplicaConfig.hpp

```diff
@@ -109,7 +109,7 @@ struct ReplicaConfig {
   bool debugPersistentStorageEnabled = false;
 
   // Messages
-  uint32_t maxExternalMessageSize = 65536;
+  uint32_t maxExternalMessageSize = 131072;
```

Comment:
This param is also defined in the configuration file, may override this value I think.

Reply:
which use case requires the increase ? just for general knowledge

Reply:
External messages are basically all the BFT messages sent between replicas + Client[Request/Reply]Msg.
In this case the ViewChangeMsg's size has changed, but I guess it could be the max size for ClientRequestMsg...